### ISSUE TITLE
Quick Unlock is not available in this database

### ISSUE THREAD

**Issue by gurgle69** [JanisEst/KeePassQuickUnlock#32]
Hi I am having a problem with Quick Unlock. I have configured everything as it says in he manual and tried all kinds of variations of settings but I always get the error “Quick Unlock is not available in this database”. Even though i can select it in the key file meny.

Does anybody know a fix?

**Comment by JanisEst** [comment]
Do you close KeePass? (= no KeePass is running anymore)

**Comment by gurgle69** [comment]
Thanks for responding so quick.

Yes i did close keepass. Is there away for it to work eaven if i cloed kee pass since last time i used the database?

**Comment by JanisEst** [comment]
No, the quick unlock informations are stored in secured process memory which gets deleted when the process closes.
To support quick unlock for a new KeePass instance the data needs to be written to a persistent storage. There it could simply be copied and used for a bruteforce attack because it may be simpler to crack than the normal KeePass master key. The same problem is present for the data in memory but it is much easier to copy a file than to dump protected memory.
